docs/robots/outdoor_robots/husky/husky.mdx

```diff
@@ -15,25 +15,25 @@ documentation.
   <div class="imgrow">
     <div class="imgcolumn3">
       <figure>
-        <a href="/docs/robots/outdoor_robots/husky/a200/user_manual_husky">
+        <a href="husky/a200/user_manual_husky">
           <img
             src="/img/website_images/husky_thumbnail.png"
             width="400"
             
           />
         </a>
-        <figcaption><a href="/docs/robots/outdoor_robots/husky/a200/user_manual_husky">Husky A200</a></figcaption>
+        <figcaption><a href="husky/a200/user_manual_husky">Husky A200</a></figcaption>
       </figure>
     </div>
     <div class="imgcolumn3">
       <figure>
-        <a href="/docs/robots/outdoor_robots/husky/a300/user_manual_husky">
+        <a href="husky/a300/user_manual_husky">
           <img
             src="/img/website_images/husky_a300_thumbnail.png"
             width="400"
           />
         </a>
-        <figcaption><a href="/docs/robots/outdoor_robots/husky/a300/user_manual_husky">Husky 300</a></figcaption>
+        <figcaption><a href="husky/a300/user_manual_husky">Husky 300</a></figcaption>
```

Comment:
Should this be updated to Husky A300 and not just Husky 300?

Reply:
Good catch, fixed!